### PR TITLE
[DOC] Tutorial for semantic segmentation

### PR DESCRIPTION
**Pull request by supersoob**
Tutorial for semantic segmentation

### CODE REVIEW

File: docs/source/guide/tutorials/base/how_to_train/semantic_segmentation.rst [hunk 1]
```diff
@@ -1,2 +1,238 @@
...
+  ...
+
+  INFO:nncf:Loaded 5286/5286 parameters
+  2023-02-21 18:09:56,134 | INFO : run task done.
+  2023-02-21 18:09:57,807 | INFO : called evaluate()
+  2023-02-21 18:09:57,807 | INFO : Computing mDice
+  2023-02-21 18:09:58,508 | INFO : mDice after evaluation: 0.9659400544959128
+  Performance(score: 0.9659400544959128, dashboard: (1 metric groups))
+
+The optimization time relies on the hardware characteristics, for example on 1 GeForce 3090 and Intel(R) Core(TM) i9-10980XE it took about 15 minutes.
```
Comment (JihwanEom) [inline docs/source/guide/tutorials/base/how_to_train/semantic_segmentation.rst]:
```suggestion
The optimization time relies on the hardware characteristics, for example on 1 NVIDIA GeForce RTX 3090 and Intel(R) Core(TM) i9-10980XE it took about 15 minutes.
```

Comment (supersoob) [reply docs/source/guide/tutorials/base/how_to_train/semantic_segmentation.rst]: I would change but seems like align it with classification and detection.

Comment (JihwanEom) [reply docs/source/guide/tutorials/base/how_to_train/semantic_segmentation.rst]: Classification and detection are wrong. Official name is `NVIDIA GeForce RTX 3090`.

Comment (supersoob) [reply docs/source/guide/tutorials/base/how_to_train/semantic_segmentation.rst]: I didn't say this is wrong. I meant cls and det should be changed too..

File: docs/source/guide/tutorials/base/how_to_train/semantic_segmentation.rst [hunk 2]
```diff
@@ -1,2 +1,238 @@
...
+  +--------------+-----------------------------------------------------+--------------------+--------------------------------------------------------------------------+
+  | SEGMENTATION |    Custom_Semantic_Segmentation_Lite-HRNet-18_OCR   |   Lite-HRNet-18    |   otx/algorithms/segmentation/configs/ocr_lite_hrnet_18/template.yaml    |
+  | SEGMENTATION | Custom_Semantic_Segmentation_Lite-HRNet-18-mod2_OCR | Lite-HRNet-18-mod2 | otx/algorithms/segmentation/configs/ocr_lite_hrnet_18_mod2/template.yaml |
+  | SEGMENTATION |  Custom_Semantic_Segmentation_Lite-HRNet-s-mod2_OCR | Lite-HRNet-s-mod2  | otx/algorithms/segmentation/configs/ocr_lite_hrnet_s_mod2/template.yaml  |
+  | SEGMENTATION |  Custom_Semantic_Segmentation_Lite-HRNet-x-mod3_OCR | Lite-HRNet-x-mod3  | otx/algorithms/segmentation/configs/ocr_lite_hrnet_x_mod3/template.yaml  |
+  +--------------+-----------------------------------------------------+--------------------+--------------------------------------------------------------------------+
+
+Note that we do not attach OCR head for supported models in default. We remain the suffix '_OCR' in ID just for backward compatibility.
```
Comment (sungmanc) [inline docs/source/guide/tutorials/base/how_to_train/semantic_segmentation.rst]: Emphasizing this sentence is more beneficial to users.

"Note that we do not attach OCR head for supported models in default."

**Review by sungmanc — DISMISSED** [review]
LGTM

File: docs/source/guide/tutorials/base/how_to_train/semantic_segmentation.rst (hunk 2 above)
Comment (kprokofi) [inline docs/source/guide/tutorials/base/how_to_train/semantic_segmentation.rst]:
```suggestion
.. note::

    We do not attach OCR head for supported models in default. We remain the suffix '_OCR' in ID just for backward compatibility.
```

File: docs/source/guide/tutorials/base/how_to_train/semantic_segmentation.rst [hunk 3]
```diff
@@ -1,2 +1,238 @@
...
+To have a specific example in this tutorial, all commands will be run on the :ref:`Lite-HRNet-18-mod2 <semantic_segmentation_models>`  model. It's a light model, that achieves competitive accuracy while keeping the inference fast.
+
+2.  Next, we need to create train/validation sets. OTX supports auto-split functionality for the semantic segmentation.
+
+Let's prepare an OTX semantic segmentation workspase running the following command:
+
+.. note::
+
+  Currently, OTX supports auto-split only for public VOC dataset format in semantic segmentation. We should specify the validation roots in argument '--val-data-roots' when using custom segmentation dataset. About custom segmentation dataset, please refer to the :doc:`advanced tutorial for datumaro <../../advanced/datumaro>`.
```
Comment (kprokofi) [inline docs/source/guide/tutorials/base/how_to_train/semantic_segmentation.rst]:
```suggestion
  Currently, OTX supports auto-split only for public VOC dataset format in semantic segmentation. We should specify the validation roots in argument '--val-data-roots' when using other supported segmentation dataset. About dataset formats for semantic segmentation, please refer to the :doc:`explanation section <../../../explanation/algorithms/segmentation/semantic_segmentation>`.
```

File: docs/source/guide/tutorials/base/how_to_train/semantic_segmentation.rst [hunk 4]
```diff
@@ -1,2 +1,238 @@
...
+Validation
+***********
+
+1. ``otx eval`` runs evaluation of a trained
+model on a specific dataset.
+
+The eval function receives test annotation information and model snapshot, trained in the previous step.
+Please note, ``label_schema.json`` file contains meta-information about the dataset and it should be located in the same folder as the model snapshot.
+
+``otx eval`` will output a mDice score for semantic segmentation.
```
Comment (kprokofi) [inline docs/source/guide/tutorials/base/how_to_train/semantic_segmentation.rst]:
```suggestion
``otx eval`` will output a ``mDice`` score for semantic segmentation.
```

File: docs/source/guide/tutorials/base/how_to_train/semantic_segmentation.rst [hunk 5]
```diff
@@ -1,2 +1,238 @@
...
+  (otx) ...$ otx export --load-weights models/weights.pth \
+                        --save-model-to openvino_model
+
+  ...
+
+  2023-02-02 03:23:03,057 | INFO : run task done.
+  2023-02-02 03:23:03,064 | INFO : Exporting completed
+
+
+3. We can check the mDice score of the IR model and the consistency between the exported model and the PyTorch model,
```
Comment (kprokofi) [inline docs/source/guide/tutorials/base/how_to_train/semantic_segmentation.rst]:
```suggestion
3. We can check the ``mDice`` score of the IR model and the consistency between the exported model and the PyTorch model,
```

**Review by kprokofi — COMMENTED** [review]
LGTM! Thank you very much. I left some comments, please, reflect them

File: docs/source/guide/tutorials/base/how_to_train/semantic_segmentation.rst (hunk 2 above)
Comment (supersoob) [reply docs/source/guide/tutorials/base/how_to_train/semantic_segmentation.rst]: I made it NOTE!!

**Review by kprokofi — APPROVED** [review]

**Review by JihwanEom — APPROVED** [review]

**Pull request merged**